### ISSUE TITLE
local client install instructions not currently working

### ISSUE THREAD

**Issue by amcasari** [LABHR/octohatrack#115]
Client output when attempting to run locally:

```
amcasari-macbookpro% python3 -m octohatrack LABHR/octohatrack
amcasari-macbookpro% pip install octohatrack
Requirement already satisfied: octohatrack in /Users/amcasari/repos-mine/octohatrack (1.0.0a0)
Requirement already satisfied: requests in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from octohatrack) (2.24.0)
Requirement already satisfied: gitpython in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from octohatrack) (3.1.11)
Requirement already satisfied: requests-cache in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from octohatrack) (0.5.2)
Requirement already satisfied: idna<3,>=2.5 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from requests->octohatrack) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from requests->octohatrack) (1.25.10)
Requirement already satisfied: certifi>=2017.4.17 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from requests->octohatrack) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from requests->octohatrack) (3.0.4)
Requirement already satisfied: gitdb<5,>=4.0.1 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from gitpython->octohatrack) (4.0.5)
Requirement already satisfied: smmap<4,>=3.0.1 in /Users/amcasari/.pyenv/versions/3.8.6/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython->octohatrack) (3.0.4)
WARNING: You are using pip version 20.2.4; however, version 20.3.1 is available.
You should consider upgrading via the '/Users/amcasari/.pyenv/versions/3.8.6/bin/python3.8 -m pip install --upgrade pip' command.
amcasari-macbookpro% octohatrack --help
usage: octohatrack [-h] [--no-cache] [--wait-for-reset] [-v] username/repo

positional arguments:
  username/repo     the name of the repo to parse

optional arguments:
  -h, --help        show this help message and exit
  --no-cache        Disable local caching of API results
  --wait-for-reset  Enable waiting for rate limit reset rather than erroring
  -v, --version     show program's version number and exit
amcasari-macbookpro% python3 -m octohatrack amcasari/octohatrack
amcasari-macbookpro% octohatrack amcasari/octohatrack

Checking repo exists....Repo does not exist: amcasari/octohatrack
```

**Comment by glasnt** [comment]
Fixed the module issue in a7b7a1b

**Comment by glasnt** [comment]
Fixed the token issue in 2e843133d72eb652b291fb6504e752083948369d

Issue persists where my rate limit handling is broken (trying to re-run the same command without cache enabled will eventually fail)

**Comment by glasnt** [comment]
see new issue for different error
